ecs-logs-collector.sh

```diff
@@ -496,6 +499,51 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/docker-sysconfig
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+  fi
+
+ try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker-storage "${info_system}"/docker/docker-sysconfig-storage
+    ok
+  else
+    info "/etc/sysconfig/docker-storage not found"
+  fi
+}
+
+
+get_docker_daemon_json(){
+  try "Gathering Docker daemon.json"
+
+  if [ -e /etc/docker/daemon.json ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/docker/daemon.json "${info_system}"/docker/daemon.json
+    ok
+  else
+    info "/etc/docker/daemon.json not found"
+  fi
+}
+
+get_docker_systemd_config(){
+  try "Gather Docker Daemon Service file"
+  
```

Comment:
Probably should have a `mkdir -p "${info_system}/docker"` here as well.

Reply:
done